views/tables/AdminUsersTable/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import Settings from '@mui/icons-material/Settings'
+import Clear from '@mui/icons-material/Clear'
+import LockOpen from '@mui/icons-material/LockOpen'
+import IconButton from '@mui/material/IconButton'
+import Checkbox from '@mui/material/Checkbox'
```

Comment:
```suggestion
import {Settings, Clear, LockOpen} from '@mui/icons-material
import {IconButton, Checkbox} from '@mui/material'
```
It's a nit but also mui suggests to import components like this

Reply:
1aa3d6f